content/courses/connecting-to-offchain-data/oracles.md

```diff
@@ -613,139 +616,144 @@ use {
-shown when you run `anchor keys list`.
+Next, replace the program ID in `lib.rs` and `Anchor.toml` by running command
+`anchor keys sync`.
 
-Next, add the following to the bottom of your Anchor.toml file. This will tell
+Next, add the following to the bottom of your `Anchor.toml` file. This will tell
 Anchor how to configure our local testing environment. This will allow us to
 test our program locally without having to deploy and send transactions to
 devnet.
 
 At the bottom of `Anchor.toml`:
 
-```toml
-[test.validator]
-url="https://api.devnet.solana.com"
-
+```toml filename="Anchor.toml"
 [test]
-startup_wait = 10000
+startup_wait = 5000
+shutdown_wait = 2000
+upgradeable = false
+
+[test.validator]
+bind_address = "0.0.0.0"
+url = "https://api.devnet.solana.com"
+ledger = ".anchor/test-ledger"
+rpc_port = 8899
 
-[[test.validator.clone]] # sbv2 devnet programID
+[[test.validator.clone]] # switchboard-solana devnet programID
 address = "SW1TCH7qEPTdLsDHRgPuMQjbQxKdH2aBStViMFnt64f"
 
-[[test.validator.clone]] # sbv2 devnet IDL
+[[test.validator.clone]] # switchboard-solana devnet IDL
 address = "Fi8vncGpNKbq62gPo56G4toCehWNy77GgqGkTaAF5Lkk"
 
-[[test.validator.clone]] # sbv2 SOL/USD Feed
-address="GvDMxPzN1sCj7L26YDK2HnMRXEQmQ2aemov8YBtPS7vR"
+[[test.validator.clone]] # switchboard-solana SOL/USD Feed
+address = "GvDMxPzN1sCj7L26YDK2HnMRXEQmQ2aemov8YBtPS7vR"
 ```
 
-Additionally, we want to import the `switchboard-v2` crate in our `Cargo.toml`
-file. Make sure your dependencies look as follows:
+Additionally, we want to import the `switchboard-solana` crate in our
+`Cargo.toml` file. Make sure your dependencies look as follows:
 
-```toml
+```toml filename="Cargo.toml"
 [dependencies]
-anchor-lang = "0.28.0"
-switchboard-v2 = "0.4.0"
+anchor-lang = "0.30.1"
+switchboard-solana = "0.30.4"
 ```
 
-Before we get started with the logic, let’s go over the structure of our
-program. With small programs, it’s very easy to add all of the smart contract
-code to a single `lib.rs` file and call it a day. To keep it more organized
-though, it’s helpful to break it up across different files. Our program will
-have the following files within the `programs/src` directory:
-
-`/instructions/deposit.rs`
-
-`/instructions/withdraw.rs`
-
-`/instructions/mod.rs`
-
-`errors.rs`
-
-`state.rs`
-
-`lib.rs`
+Before diving into the program logic, let's review the structure of our smart
+contract. For smaller programs, it's tempting to put all the code in a single
+`lib.rs` file. However, organizing the code across different files helps
+maintain clarity and scalability. Our program will be structured as follows
+within the `programs/burry-escrow` directory:
+
+```sh
+└── burry-escrow
+    ├── Cargo.toml
+    ├── Xargo.toml
+    └── src
+        ├── constants.rs
+        ├── errors.rs
+        ├── instructions
+        │   ├── deposit.rs
+        │   ├── mod.rs
+        │   └── withdraw.rs
+        ├── lib.rs
+        └── state.rs
+```
```

Comment:
No feedback here just wanted to say I really appreciate the attention to detail, as always.